### PR TITLE
Update attribution page, include examples

### LINKED ISSUE #1647: Add examples to license attribution
## Issue description

Users are having a hard time understanding what we want our attribution to look like

#### What page is the problem visible

https://gis.utah.gov/about/policy/license-disclaimer/ or
https://gis.utah.gov/about/media/

#### Expected result

I expected to see some examples for common items, maps, blogs, books, etc.

#### Actual result

Instead I have to read an interpret and guess. Give me the easy button.

### PR DESCRIPTION
**Pull request by jacobdadams**
I've updated the attribution page to be more clear and (hopefully) in-line with the CC BY 4.0 license. Feedback welcome.

Closes #1647.

### CODE REVIEW

File: about/policy/license-disclaimer/index.md [hunk 1]
```diff
@@ -24,18 +29,38 @@ The data and services provided by UGRC is licensed under a [Creative Commons Att
 ### License Attribution
 {: .text-left}
 
-UGRC requests that where possible and practical, **appropriate attribution** shall be made by users of this website, data, or any other services provided by UGRC with the following criteria:
+UGRC requests that wherever possible and practical, you include **appropriate attribution** when using data or services provided by UGRC in your maps, web applications, or other content. This includes providing credit to UGRC and indicating you made any changes to the data.
```
Comment (gregbunce) [inline about/policy/license-disclaimer/index.md:32]: nice work on these examples.  i guess the BY is what's making our license a little more complex.

File: about/policy/license-disclaimer/index.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: State Geographic Information Database License and Disclaimer
+title: Utah Geospatial Resource Center Data License and Disclaimer
```
Comment (steveoh) [inline about/policy/license-disclaimer/index.md:2]: **praise**: this is a good change.

File: about/policy/license-disclaimer/index.md (hunk 1 above)
Comment (steveoh) [reply about/policy/license-disclaimer/index.md:32]: I think our language of wherever possible and practical loosens it up. We could list that we are open to exceptions?

File: about/policy/license-disclaimer/index.md [hunk 3]
```diff
@@ -24,18 +29,38 @@ The data and services provided by UGRC is licensed under a [Creative Commons Att
 ### License Attribution
 {: .text-left}
 
-UGRC requests that where possible and practical, **appropriate attribution** shall be made by users of this website, data, or any other services provided by UGRC with the following criteria:
+UGRC requests that wherever possible and practical, you include **appropriate attribution** when using data or services provided by UGRC in your maps, web applications, or other content. This includes providing credit to UGRC and indicating you made any changes to the data.
+
+For referenced data hosted by other organizations (such as data from the Utah Geological Survey), please contact the owner for attribution guidelines.
```
Comment (steveoh) [inline about/policy/license-disclaimer/index.md:34]: **question**: should referenced be reference? Referenced or reference seems superfluous or what am I missing. Maybe I'm tired.

File: about/policy/license-disclaimer/index.md [hunk 4]
```diff
@@ -24,18 +29,38 @@ The data and services provided by UGRC is licensed under a [Creative Commons Att
...
+
+**Detailed references** in a paper or book should follow your particular style guide. Please include the last updated date if it is available in the metadata or the date you accessed/downloaded the data if not. For example, this is what it could look like in an MLA-esque format:
+
+- State Geographic Information Database. _Utah Address Points_. (Feature class). Salt Lake City, UT: Utah Geographic Resource Center, July 2021. Geospatial point data. 27 July 2021. https://gis.utah.gov/data/location/address-data/.
```
Comment (steveoh) [inline about/policy/license-disclaimer/index.md:38]: **praise**: this is cool!

**Review by steveoh — DISMISSED** [review]

File: about/policy/license-disclaimer/index.md (hunk 4 above)
Comment (steveoh) [reply about/policy/license-disclaimer/index.md:38]: https://twitter.com/natfriedman/status/1420122675813441540?s=20 should we add somethign like this to our website repo?

File: about/policy/license-disclaimer/index.md (hunk 3 above)
Comment (jacobdadams) [reply about/policy/license-disclaimer/index.md:34]: Yeah, it's a little left-field. It's (trying not to say "referencing" here...) a tie-in to the definitions used in the sharing agreement, where "hosted" = data we host in the internal/Open SGID and "referenced" = other people's data referenced via the SGID Index or Open Data.

File: about/policy/license-disclaimer/index.md (hunk 4 above)
Comment (jacobdadams) [reply about/policy/license-disclaimer/index.md:38]: Gotta have something to show for all those college papers :)

**Comment by jacobdadams** [comment]
>I think our language of wherever possible and practical loosens it up. We could list that we are open to exceptions?

It's not letting me reply directly, but how does this latest change look to you?

**Comment by steveoh** [comment]
Yah that looks fine to me

**Review by steveoh — APPROVED** [review]

**Pull request merged**